### DISCUSSION TITLE
朋友们拿到原画集了么

### DISCUSSION THREAD

**Discussion by canyusuo** [share121/inter-knot#5323] [崩坏3]
<img width="1280" alt="Autodesk Flow Design Screenshot 2024 09 23 - 12 06 53 98" src="https://github.com/user-attachments/assets/ba5fc612-67b2-44eb-8cff-dd8326de4b51">
<img width="1280" alt="Autodesk Flow Design Screenshot 2024 09 23 - 12 07 29 67" src="https://github.com/user-attachments/assets/3134784f-15a6-4c9a-959d-794ac23b493e">
嘻嘻，第一梯队已经拿到了<img src="https://img-static.mihoyo.com/communityweb/upload/59353722c9fb0c5d926e84638b044e02.gif" alt="爱了爱了" width="75" >

**Comment by aomengxin** [comment]
萌新哭哭

**Comment by FAIR-MOONLIGHT** [comment]
第三万八，早上没想起来，看到群友说才想起来
